source/progress.ts

```diff
@@ -1,48 +0,0 @@
-import EventEmitter = require('events');
-import {Transform as TransformStream} from 'stream';
-import is from '@sindresorhus/is';
-
-export function createProgressStream(name: 'downloadProgress' | 'uploadProgress', emitter: EventEmitter, totalBytes?: number | string): TransformStream {
```

Comment:
This has been removed in favor of the Duplex stream (yeah, we can do things like this there).